test/e2e/storage/flexvolume.go

```diff
@@ -0,0 +1,156 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package storage
+
+import (
+	"fmt"
+	"math/rand"
+	"net"
+	"path"
+
+	. "github.com/onsi/ginkgo"
+	"k8s.io/api/core/v1"
+	"k8s.io/kubernetes/pkg/client/clientset_generated/clientset"
+	"k8s.io/kubernetes/test/e2e/framework"
+	"k8s.io/kubernetes/test/e2e/generated"
+)
+
+const (
+	sshPort                = "22"
+	examplesDir            = "examples/volumes/flexvolume/"
+	defaultVolumePluginDir = "/usr/libexec/kubernetes/kubelet-plugins/volume/exec"
+	// On gce, root may be read-only and controller-manager containerized. Assume
+	// controller-manager has started with --flex-volume-plugin-dir equal to this
+	// (see cluster/gce/config-test.sh and function mount-master-pd)
```

Comment:
@verult may be making changes in this area for defining the default flex volume install path for gci images.